ui/components/ui/tooltip/index.scss

```diff
@@ -1,30 +1,44 @@
-.tippy-tooltip.white-theme {
-  background: var(--color-background-default);
-  color: var(--color-text-default);
-  box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
-  padding: 12px 16px;
-  padding-bottom: 11px;
-
-  .tippy-tooltip-content {
-    @include H7;
-
-    text-align: left;
-    color: var(--color-text-alternative);
+.tippy-popper {
+  .tippy-tooltip.tippy-tooltip--mm-custom-theme {
+    background: var(--color-background-default);
+    color: var(--color-text-default);
+    box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+    padding: 12px 16px;
+    padding-bottom: 11px;
+
+    .tippy-tooltip-content {
+      @include H7;
+
+      text-align: left;
+      color: var(--color-text-alternative);
+
+      a {
+        color: var(--color-primary-default);
+      }
+
+      p {
+        margin-bottom: 16px;
+
+        &:last-child {
+          margin-bottom: 0;
+        }
+      }
+    }
   }
-}
 
-.tippy-popper[x-placement^=top] .white-theme [x-arrow] {
-  border-top-color: var(--color-background-default);
-}
+  &[x-placement^=top] .tippy-tooltip.tippy-tooltip--mm-custom-theme [x-arrow] {
+    border-top-color: var(--color-background-default);
+  }
 
-.tippy-popper[x-placement^=right] .white-theme [x-arrow] {
-  border-right-color: var(--color-background-default);
-}
+  &[x-placement^=right] .tippy-tooltip.tippy-tooltip--mm-custom-theme [x-arrow] {
+    border-right-color: var(--color-background-default);
+  }
 
-.tippy-popper[x-placement^=left] .white-theme [x-arrow] {
-  border-left-color: var(--color-background-default);
-}
+  &[x-placement^=left] .tippy-tooltip.tippy-tooltip--mm-custom-theme [x-arrow] {
+    border-left-color: var(--color-background-default);
+  }
 
-.tippy-popper[x-placement^=bottom] .white-theme [x-arrow] {
-  border-bottom-color: var(--color-background-default);
+  &[x-placement^=bottom] .tippy-tooltip.tippy-tooltip--mm-custom-theme [x-arrow] {
+    border-bottom-color: var(--color-background-default);
+  }
 }
```

Comment:
The meat and potatoes of this PR. Updating the Tooltip styles and custom theme name